build.gradle

```diff
@@ -216,8 +208,14 @@ private def colorize(String color, String message) {
 }
 
 repositories {
-    mavenLocal()
+//    mavenLocal() // uncomment for testing with local maven
     mavenCentral()
+    maven {
+        url 'https://repository.jboss.org/nexus/content/repositories/snapshots'
+    }
+    maven {
+        url 'https://repository.jboss.org/nexus/content/groups/public'
```

Comment:
how's this repo different from the existing one at https://repository.jboss.org?

Reply:
You are right, we dont need it, it was in intellij-common-blabla manual, so I added it.

Reply:
I`ll remove it in the next commit.